skore/src/skore/sklearn/_plot/metrics/precision_recall_curve.py

```diff
@@ -910,3 +912,68 @@ def _compute_data_for_display(
             ml_task=ml_task,
             report_type=report_type,
         )
+
+    def frame(self, with_average_precision: bool = False) -> DataFrame:
+        """Get the data used to create the precision-recall curve plot.
+
+        Parameters
+        ----------
+        with_average_precision : bool, default=False
+            Whether to include the average precision column in the returned DataFrame.
+
+        Returns
+        -------
+        DataFrame
+            A DataFrame containing the precision-recall curve data with columns
+            depending on the report type:
+
+            - estimator_name: Name of the estimator (when comparing estimators)
+            - split_index: Cross-validation fold ID (when doing cross-validation)
+            - label: Class label (for multiclass-classification)
+            - threshold: Decision threshold
+            - precision: Precision score at threshold
+            - recall: Recall score at threshold
+            - average_precision: average precision (when `with_average_precision=True`)
```

Comment:
Since they are parameters, let's them look like this in the documentation.

```suggestion
            - `estimator_name`: Name of the estimator (when comparing estimators)
            - `split_index`: Cross-validation fold ID (when doing cross-validation)
            - `label`: Class label (for multiclass-classification)
            - `threshold`: Decision threshold
            - `precision`: Precision score at threshold
            - `recall`: Recall score at threshold
            - `average_precision`: average precision (when `with_average_precision=True`)
```